README.md

```diff
@@ -48,10 +49,55 @@ const HomePage = () => {
     i18n.locale('de'); // if dict was already loaded
     i18n.locale('de', DE); // set and load dict at the same time
 
-    // Get current locale
-    i18n.locale(); // de
-
     // Checkout https://github.com/lukeed/rosetta for full interpolation support
     return <p>{i18n.t('welcome', { username })}</p>;
 };
 ```
+
+### Usage with `getStaticProps`
+
+You can use Next.js's static APIs to feed your `_app.js`'s `lngDict`:
+
+```js
+import { I18n } from 'next-localization';
+
+export default function MyApp({ Component, pageProps }) {
+    return (
+        <I18n lngDict={pageProps.lngDict} locale={pageProps.lng}>
+            <Component {...pageProps} />
+        </I18n>
+    );
+}
+```
+
+Each page should define `getStaticProps` and `getStaticPaths`:
+
+```js
+// pages/[lng]/index.js
+
+export const getStaticProps = async ({ params }) => {
+  // translations obtained from external API at build time
+  // this example is fetching page `index` once for each language defined in `params.lng`
+  const lngDict = await getTranslations('index', params?.lng);
```

Comment:
`getAvailableLanguages` and `getTranslations` aren't defined here it could cause to much confusion. You don't think the example project isn't enough?

Reply:
I think pointing to the hooks in the example project is sufficient and easier to maintain

Reply:
Goods docs are better than an example repo IMO. If you'd rather have these changes in the readme, I understand and feel free to close the PR.

I can also change the example to use JSON files from file system like in the example repo if you want.

Reply:
Thanks for the PR but I'd like to keep the readme simple.